src/Extensions/Logging/EmptyMachineInformation.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using System.Reflection;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// Base class for machine information
+	/// </summary>
+	public class EmptyMachineInformation : IMachineInformation
+	{
+		/// <summary>
+		/// Create instance of machine information
+		/// </summary>
+		/// <remarks>
+		/// Please keep this initialization as lightweight as possible
+		/// </remarks>
+		public EmptyMachineInformation()
+		{
+			MachineName = DefaultEmptyValue;
```

Comment:
Move those to the property:
MachineName {...} = DefaultEmptyValue

Reply:
Why it would be better? I like having all initialization in one place

Reply:
If you add a new ctor you'd need to copy paste all this

Reply:
Or I can just call this one from the new constructor, like:
```
EmptyMachineInformation(SomeNewInfo info) : this()
{
}
```